robottelo/constants.py

```diff
@@ -155,13 +155,21 @@
     'sha1': "sha1",
 }
 
+REPO_TAB = {
+    'RPMs': "RPMs",
```

Comment:
nitpick: keys in lowercase? Otherwise i can't see advantages in such dict with same keys/values.